guides/source/command_line.md

```diff
@@ -263,7 +263,15 @@ Usage:
 ActiveRecord options:
       [--migration], [--no-migration]        # Indicates when to generate migration
                                              # Default: true
-
+      [--timestamps], [--no-timestamps]      # Indicates when to generate timestamps
+                                             # Default: true
+      [--parent=PARENT]                      # The parent class for the generated model
+      [--indexes], [--no-indexes]            # Add indexes for references and belongs_to columns
+                                             # Default: true
+      [--primary-key-type=PRIMARY_KEY_TYPE]  # The type for primary key
+  --db, [--database=DATABASE]                # The database for your model's migration. By default, the current environment's primary database is used.
+  -t, [--test-framework=NAME]                # Test framework to be invoked
+                                             # Default: test_unit
```

Comment:
I would actually be inclined to omit all these options here, [the way we do for `generate controller`](https://github.com/rails/rails/blame/23a9e29d9b1882e4d28530fca98b8c57b3de0410/guides/source/command_line.md#L182-L188).

Reply:
I'll remove it but I feel documenting all the options actual helps when someone is looking to skip a timestamp or use `uuid` as primary-key-type and Rails guide should speak more about these options. Similar should happen with controllers as well. Just a thought 😬